2-mandatory/3-shopping-list.js

```diff
@@ -43,11 +52,18 @@ test("createShoppingList works for pancakes recipe", () => {
 test("createShoppingList works for margherita pizza recipe", () => {
   let recipe2 = {
     name: "margherita pizza",
-    ingredients: ["flour", "salt", "yeast", "tinned tomatoes", "oregano", "mozarella"],
+    ingredients: [
+      "flour",
+      "salt",
+      "yeast",
+      "tinned tomatoes",
+      "oregano",
+      "mozarella",
+    ],
```

Comment:
You know what I'm going to say here :)
Try not to change the parts of the code you're not asked to change to keep the commits focused on the task at hand, and not formatting changes for the code you're given.
If you think that code needs to be changed for readability (and in this case I agree) do it in a separate commit labeled "code cleanup" or something like that. And make a real PR so that the source repository gets fixed!